src/components/transaction-list/TransactionList.js

```diff
@@ -146,54 +154,38 @@ const TransactionList = ({
     Clipboard.setString(accountAddress);
   };
 
-  const onCopyTooltipPress = () => {
-    Clipboard.setString(accountENS || accountAddress);
-  };
-
-  const formatAddress = address => {
-    if (address) {
-      return abbreviations.address(
-        address,
-        4,
-        abbreviations.defaultNumCharsPerSection
-      );
-    }
-    return '';
-  };
+  const onAccountNamePress = useCallback(() => {
+    navigate(Routes.CHANGE_WALLET_SHEET);
+  }, [navigate]);
 
   if ((!initialized && !navigation.isFocused()) || isLoading) {
-    return (
-      <View style={isMultiwalletAvailable ? { marginTop: 20 } : null}>
-        <LoadingState>{header}</LoadingState>
-      </View>
-    );
+    return <LoadingState>{header}</LoadingState>;
   }
 
-  const addressOrEns = accountENS || formatAddress(accountAddress);
-
   const data = {
     requests,
     transactions,
   };
 
   return (
-    <View style={style}>
+    <Row flex={1}>
       <NativeTransactionListView
-        accountAddress={addressOrEns}
+        accountAddress={accountName}
         accountColor={colors.avatarColor[accountColor]}
-        accountName={accountName}
+        accountName={accountEmoji}
         addCashAvailable={addCashAvailable}
         data={data}
+        emoji={accountEmoji}
```

Comment:
is emoji actually used in native txn list? (also, perhaps should we rename accountName / emoji on the native txn list so it isn't too confusing)

Reply:
good call. Not being used and renamed to accountSymbol instead.

Reply:
it seems like multiwallet breaks the avatar builder, which I think is where the emoji is used

Reply:
Actually I added that flag and forgot to remove it.  
But yeah, there's some work needed on the avatar builder after this, which I think we should do after and not now since the flag has been off.